cmd/main.go

```diff
@@ -209,7 +205,7 @@ func initFlagsAndEnv(fs *pflag.FlagSet) {
 	ProxmoxTokenID = env.GetString("PROXMOX_TOKEN", "")
 	ProxmoxSecret = env.GetString("PROXMOX_SECRET", "")
 
-	fs.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	fs.StringVar(&metricsAddr, "diagnostics-address", ":8080", "The address the metric endpoint binds to.")
```

Comment:
why? this is a breaking change.

Reply:
Controller runtime introduced this change.
In our case, it will not be a BREaking change,  as this will be part of the deployment manifest.

Reply:
https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.16.0

Reply:
It's recommended to follow the controller-runtime schema

Reply:
Well, i think i will roll it back

Reply:
This project is still in development, so breaking changes in order to conform new standards should not be avoided. If you switch to diagnostics-address you should also add the option to serve securely